src/components/auth/dataUpload.js

```diff
@@ -0,0 +1,143 @@
+import React, {useState} from 'react';
+import { axiosWithAuth } from "../../utils/axiosWithAuth.js";
+import { ifDev } from "../../utils/removeAttribute.js";
+
+import IMDbImg from '../auth/images 2.png'
+import LetterboxdImg from '../auth/Letterboxd_logo_(2018) 3.png'
+import './dataUpload.scss'
+
+const DataUpload = props => {
+
+
+const [input, setInput] = useState({file:''})
+    const [ratings, setRatings] = useState({})
+    const [loading, setLoading] = useState()
+
+  
+
+
+function clikL() {
+    document.getElementById("myDropdownL").classList.toggle("show");
+  }
+
+function clikR() {
+    document.getElementById("myDropdownR").classList.toggle("show");
+  }  
+
+  window.onclick = function(event) {
```

Comment:
Curious why you decided to write in on the entire window.

Reply:
If you click anything but the buttons it will close the open dropdown, to that point Ben maybe something like clicking the letterboxd closes the imdb dropdown would be the final touch?